hw02-generics-collections/src/main/java/ru/otus/kovaleva/CustomerReverseOrder.java

```diff
@@ -0,0 +1,17 @@
+package ru.otus.kovaleva;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class CustomerReverseOrder {
+
+    private Deque<Customer> customers = new ArrayDeque<>();
```

Comment:
Можно добавить `final`:
```suggestion
    private final Deque<Customer> customers = new ArrayDeque<>();
```